paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/KlarnaCountrySpec.kt

```diff
@@ -0,0 +1,9 @@
+package com.stripe.android.paymentsheet.elements
+
+/**
+ * This is the specification for a klarna country field
+ * @property onlyShowCountryCodes: a list of country code that should be shown.  If empty all
+ * countries will be shown.
+ */
+internal data class KlarnaCountrySpec(val onlyShowCountryCodes: Set<String> = emptySet()) :
```

Comment:
seems this is exactly the same with `CountrySpec`, is a separate file needed here?

Reply:
We need to know what currency is being used by the payment intent, so we need a separate spec for this. Michelle and I decided to just add another one until she completes her survey of payment methods and maybe we can come back and build something more general.